README.md

```diff
@@ -117,7 +119,8 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+it creates database on sql not psql
+first one creates db named tunr 2nd one doesnt have a db name.
 ```
```

Comment:
The first one creates a Rails application in a new `tunr` directory, which is placed in the current directory. It does not indicate what database management tool we're using, so it defaults to SQLite (not Postgres).

The second one creates a Rails application in the current directory -- no additional `tunr` directory. It also indicates that we do want to use Postgres as our database management tool.